includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -658,6 +720,44 @@ public function display_payment_request_button_separator_html() {
 		<?php
 	}
 
+	/**
+	 * Display payment request redirect dialog.
+	 */
+	public function display_payment_request_redirect_dialog() {
```

Comment:
Well observed. This was actually considered and discussed before (p1615518244176400/1615427515.099500-slack-CGGCLBN58).

The `Anyone can register` option doesn't really apply in this case because it's implicit that to use the PRB you'd need to have that option checked. Any issues with having it disabled would be noticeable when testing. So we don't need to change the messaging.